server/server.js

```diff
@@ -6,8 +6,32 @@
 
 // list the food trucks returned by filterByDay, passing in the current day name.
 // e.g. filterByDay(day); where "day" is determined using the enhancedDate module
-// The list of trucks returned will be an array of food truck objects. Iterate 
-// through the list, building up a string of food truck names. Once you're done 
+// The list of trucks returned will be an array of food truck objects. Iterate
+// through the list, building up a string of food truck names. Once you're done
 // iterating through that list, display the string you built up.
 
 // Remember that the response is a stream object that must be closed.
+var http = require('http');
+var enhancedDate = require('./enhancedDate');
+var trucks = require('./trucks');
+http.createServer(function (request, response) {
+  'use strict';
+  response.writeHead(200, { 'Content-Type': 'text/plain' });
+  var dayName = enhancedDate.getDayName();
```

Comment:
This is causing an error because the internal date in your module hasn't been set yet. You will either need to explicitly call enhancedDate.setDate() first or you'll have to change your implementation of getDayName() to do so.